### ISSUE TITLE
rsync c

### ISSUE THREAD

**Issue by MithrilMan** [AbdBarho/stable-diffusion-webui-docker#312]
**Has this issue been opened before?**

- [X ] It is not in the [FAQ](https://github.com/AbdBarho/stable-diffusion-webui-docker/wiki/FAQ), I checked.
- [X ] It is not in the [issues](https://github.com/AbdBarho/stable-diffusion-webui-docker/issues?q=), I searched.

**Describe the bug**

Tried to create a model with dreambooth extension but it failed with a "Cannot allocate memory (12)" error caused by **_rsync_**

![image](https://user-images.githubusercontent.com/5107375/214595688-5909247b-b621-42f1-8b7b-2ae9d2550362.png)

**Which UI**

auto

**Additional context**

I've installed the dreambooth extensions, I've created a script in startup.sh to mount dreambooth folder
When I try to create a model, during the creation it error out complaining about rsync that's causing memory error

From my understanding there is an infinite loop caused by [#309](https://github.com/AbdBarho/stable-diffusion-webui-docker/pull/309) that happens every 2 seconds and it seems causing problem to dreambooth model creation (and eventually training)

( I've 32GB of ram and a 3090TI GPU )

**Comment by AbdBarho** [comment]
@MithrilMan that is correct, I will remove it for now, but some other extensions will break.
